backend/src/app.py

```diff
@@ -16,6 +16,10 @@ def create_app():
     spotify = SpotifyClient()
     app.config.from_object(Config())
     app.config["CORS_HEADERS"] = "Content-Type"
+    app.config.update(
+        SESSION_COOKIE_SAMESITE="None",
+        SESSION_COOKIE_SECURE="True",
+    )
```

Comment:
Nice work getting this working!👍 
Minor, but this strikes me as one of those moments it might be worth a quick explanatory comments to remind yourself _why_ this has to be done